packages/terra-section-header/src/SectionHeader.scss

```diff
@@ -0,0 +1,92 @@
+        border-top: var(--terra-section-header-hover-border-top);
+        box-shadow: var(--terra-section-header-hover-box-shadow);
+        color: var(--terra-section-header-hover-color);
+      }
+
+      &:focus {
+        background: var(--terra-section-header-focus-background, #e8e9ea);
+        border-bottom: var(--terra-section-header-focus-border-bottom);
+        border-left: var(--terra-section-header-focus-border-left);
+        border-right: var(--terra-section-header-focus-border-right);
+        border-top: var(--terra-section-header-focus-border-top);
+        box-shadow: var(--terra-section-header-focus-box-shadow, 0 0 1px 3px rgba(76, 178, 233, 0.5), 0 0 7px 4px rgba(76, 178, 233, 0.35));
+        color: var(--terra-section-header-focus-color);
+      }
+
+      &:active {
+        background: var(--terra-section-header-active-background, #d7d8d9);
+        border-bottom: var(--terra-section-header-active-border-bottom);
+        border-left: var(--terra-section-header-active-border-left);
+        border-right: var(--terra-section-header-active-border-right);
+        border-top: var(--terra-section-header-active-border-top);
+        box-shadow: var(--terra-section-header-active-box-shadow);
+        color: var(--terra-section-header-active-color);
+      }
+    }
+  }
+
+  .title {
+    display: inline;
+    font-size: 1rem;
+    font-weight: var(--terra-section-header-font-weight, bold);
+    text-transform: var(--terra-section-header-text-transform);
+  } 
+
+  .accordion-icon-wrapper {
+    @include terra-margin-start(var(--terra-section-header-accordion-icon-margin-start));
+    @include terra-margin-end(var(--terra-section-header-accordion-icon-margin-end, 0.357142857rem));
+    @include terra-padding-start(var(--terra-section-header-accordion-icon-padding-start));
+    @include terra-padding-end(var(--terra-section-header-accordion-icon-padding-end));
+    padding-bottom: var(--terra-section-header-accordion-icon-padding-bottom);
+    padding-top: var(--terra-section-header-accordion-icon-padding-top, 0.142857143rem);
+  }
+
+  .accordion-icon {
+    // stylelint-disable-next-line string-quotes
+    background: var(--terra-section-header-accordion-icon, inline-svg("<svg version='1.1' id='Layer_1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 48 48' enable-background='new 0 0 48 48' xml:space='preserve'><title>0023_caretRight_a</title><path id='caretRight' d='M12,0l24,24L12,48V0z'/></svg>"));
+    background-repeat: no-repeat;
+    background-size: contain;
+    display: block;
+    height: 0.8571428571rem;
+    transform: rotate(0deg);
+    width: 0.8571428571rem;
+
+    // flips the icon about its vertical axis, presenting it in its mirrored-image form
+    [dir=rtl] & {
+      transform: scaleX(-1);
+    }
+
+    // tells the rtl postcss plugin to not transform this by default 
+    [dir] &.is-open {
```

Comment:
Does this transform correctly for open rtl icons?

Reply:
yes